dotnet/xml/ns-Microsoft.Identity.Client.xml

```diff
@@ -1,6 +1,66 @@
 <Namespace Name="Microsoft.Identity.Client">
   <Docs>
-    <summary>To be added.</summary>
-    <remarks>To be added.</remarks>
+    <summary overwrite="false">
+        <format type="text/markdown"><![CDATA[
+
+[![NuGet](https://img.shields.io/nuget/v/Microsoft.Identity.Client.svg?style=flat-square&label=Microsoft.Identity.Client&colorB=00b200)](https://www.nuget.org/packages/Microsoft.Identity.Client/)  [![NuGet downloads](https://img.shields.io/nuget/dt/Microsoft.Identity.Client?style=flat-square)](https://www.nuget.org/packages/Microsoft.Identity.Client/) [![GitHub Repo stars](https://img.shields.io/github/stars/AzureAD/microsoft-authentication-library-for-dotnet?style=flat-square)](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet)
+
+`Microsoft.Identity.Client` is the core namespace for the Microsoft Authentication Library (MSAL) for .NET, available through the [`Microsoft.Identity.Client`](https://www.nuget.org/packages/Microsoft.Identity.Client/) package. It contains all the key components that you need to acquire a token from supported authentication providers.
+
+## Installation
+
+Using the [.NET Command-Line Interface (CLI)](/dotnet/core/tools/):
+
+```bash
+dotnet add package Microsoft.Identity.Client
+```
+
+Using the [NuGet Command Line Interface (CLI)](/nuget/tools/nuget-exe-cli-reference):
+
+```bash
+nuget install Microsoft.Identity.Client
+```
+
+Using the [Package Manager Console](/nuget/tools/package-manager-console):
+
+```powershell
+Install-Package Microsoft.Identity.Client
+```
+
+Using [Visual Studio](/visualstudio/):
+
+1. Open the Solution Explorer.
+2. Right-click on a project within the solution.
+3. Click on **Manage NuGet Packages...**
+4. Click on the Browse tab and search for `Microsoft.Identity.Client`.
+5. Click on the `Microsoft.Identity.Client` package and click **Install**.
+
+## Documentation
+
+To get started, check out the [Microsoft Authentication Library for .NET](/entra/msal/dotnet/) guide.
+
+## Samples
+
+Refer to [Microsoft identity platform code samples](/azure/active-directory/develop/sample-v2-code) and the [Samples Browser](/samples/browse/?products=microsoft-authentication-library&expanded=microsoft-authentication-library).
```

Comment:
`/samples/browse/?products=microsoft-authentication-library&expanded=microsoft-authentication-library`
This link expands the MSAL filter, but it only has samples for Java and Node.

Reply:
Yeah, at some point we will include .NET there too (some point = next week).